src/services/Credential.js

```diff
@@ -506,20 +483,22 @@ class Credential {
 		return this.hasKey(key) ? (this[key] || this[key.toLowerCase()]) : null;
 	}
 
+	get hasPrivateKey() {
+		return this.hasKey("PRIVATE_KEY");
+	}
+
+	get getPrivateKey() {
+		return this.getKey("PRIVATE_KEY");
+	}
+
+	get getP7B() {
```

Comment:
`getP7B` -> `P7B`

Reply:
Property is accessible directly, so no need for the getter.